config.example.yaml

```diff
@@ -86,8 +86,11 @@ nodes:
   protocol: arc
   # list of apis used for getting and broadcasting transactions
   apis:
-    - arc_url: https://arc.gorillapool.io
-      token:
+    # gorillapool can be used as well
+#    - arc_url: https://arc.gorillapool.io
+#    - token: ""
+    - arc_url: https://tapi.taal.com/arc/v1
```

Comment:
We should use `https://tapi.taal.com/arc/v1`
without the "t" letter.

Reply:
Found endpoint here 
https://docs.taal.com/core-products/transaction-processing/arc-endpoints-beta

Reply:
Changed